framework/include/utils/TimedPrint.h

```diff
@@ -81,8 +82,8 @@ class TimedPrint final
 #define ARGS args_in...
 
 #endif
-    if (_active_instance)
-      return;
+    // if (_active_instance)
+    return;
```

Comment:
What's going on?

Reply:
Well - I was just deactivating the TimedPrints... I think I will just remove the object altogether now though.